packages/arb-ts/src/lib/abi/hardhat.d.ts

```diff
@@ -0,0 +1,431 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
Do we have a new contract named `hardhat` for this to get generated?

Reply:
Ah seems like a new typechain feature
https://github.com/ethereum-ts/TypeChain/issues/370